ci/requirements/min-all-deps.yml

```diff
@@ -10,46 +10,45 @@ dependencies:
   - python=3.8
   - boto3=1.13
   - bottleneck=1.3
-  # cartopy 0.18 conflicts with pynio
-  - cartopy=0.17
+  - cartopy=0.19
   - cdms2=3.1
   - cfgrib=0.9
-  - cftime=1.2
+  - cftime=1.4
   - coveralls
-  - dask-core=2.30
-  - distributed=2.30
-  - h5netcdf=0.8
-  - h5py=2.10
-  # hdf5 1.12 conflicts with h5py=2.10
+  - dask-core=2021.04
+  - distributed=2021.04
+  - h5netcdf=0.11
+  - h5py=3.1
+  # hdf5 1.12 conflicts with h5py=3.1
   - hdf5=1.10
   - hypothesis
   - iris=2.4
   - lxml=4.6  # Optional dep of pydap
-  - matplotlib-base=3.3
+  - matplotlib-base=3.4
   - nc-time-axis=1.2
   # netcdf follows a 1.major.minor[.patch] convention
   # (see https://github.com/Unidata/netcdf4-python/issues/1090)
   # bumping the netCDF4 version is currently blocked by #4491
   - netcdf4=1.5.3
-  - numba=0.51
-  - numpy=1.18
+  - numba=0.53
+  - numpy=1.19
   - packaging=20.0
-  - pandas=1.1
-  - pint=0.16
+  - pandas=1.2
+  - pint=0.17
   - pip
   - pseudonetcdf=3.1
   - pydap=3.2
-  - pynio=1.5
+  # - pynio=1.5.5
```

Comment:
@fulminemizzega If you can figure out a combination that works with this version of the file, we could re-enable.

We'd also be happy re-enabling it in `environment.yml` as long as it doesn't restrict other packages.